src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -2,84 +2,41 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=========================================================================
-**
-** Purpose: 
-** A complex number z is a number of the form z = x + yi, where x and y 
-** are real numbers, and i is the imaginary unit, with the property i2= -1.
-**
-===========================================================================*/
-
-using System;
 using System.Globalization;
 using System.Diagnostics.CodeAnalysis;
-using System.Diagnostics.Contracts;
 
 namespace System.Numerics
 {
+    /// <summary>
+    /// A complex number z is a number of the form z = x + yi, where x and y 
+    /// are real numbers, and i is the imaginary unit, with the property i2= -1.
+    /// </summary>
```

Comment:
FYI, the plan is that very soon the MSDN docs will be used to populate XML comments in the source.  It's not worth spending time adding such comments manually; in fact, doing so could cause the other process to take longer.